packages/pricing/src/models/price-set-money-amount.ts

```diff
@@ -61,6 +65,19 @@ export default class PriceSetMoneyAmount {
   @Property({ columnType: "text", nullable: true })
   title: string | null = null
 
+  @PriceSetMoneyAmountCurrencyCodeIndex.MikroORMIndex()
+  @Property({ columnType: "text" })
+  currency_code: string
+
+  @Property({ columnType: "numeric", serializer: Number })
+  amount: number
```

Comment:
suggestion: use BigNumber

Reply:
Already suggested and it will be part a separate pr

Reply:
Got it

Reply:
This one is coming 2 PRs down, got one more biggie before it. 😄